### PR TITLE
support for ImageOverlay layer

### PR DESCRIPTION
**Pull request by VincentDS**

### CODE REVIEW

File: addon/layers/image.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * @extends Layer
+ */
+
+export default Layer.extend({
+  imageUrl: null,
+  imageBounds: null,
+  options: null,
+
+  _newLayer: function() {
+    return L.imageOverlay(get(this,'imageUrl'), get(this,'imageBounds'), get(this, 'options'));
```
Comment (nickiaconis) [inline addon/layers/image.js]: nitpick: Please use consistent spacing after commas.

File: addon/layers/image.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  }
+});
```
Comment (nickiaconis) [inline addon/layers/image.js]: nitpick: Please include a newline at the end of files.

**Comment by nickiaconis** [comment]
Hi @VincentDS. Thanks for the contribution. Can you write test(s) for your new layer? I've also called out a few code styling nitpicks in the diff. Otherwise, this looks great! :+1:

**Comment by VincentDS** [comment]
done! :)

**Comment by nickiaconis** [comment]
Fantastic! :grinning:

**Comment by nickiaconis** [comment]
I'm not sure I agree with changing `imageUrl` to `url` for two reasons: this property is documented by Leaflet as `imageUrl`, and the tile layer class uses `tileUrl`, which is akin to using `imageUrl` for the image layer class.

`bounds` I can understand as Leaflet does not document it consistently. We also already have the idea of layer's bounds living on a property named "bounds" from the bounds mixin, so makes sense from that angle too.

**Pull request merged**